recipes/openblas/bld.bat

```diff
@@ -0,0 +1,9 @@
+
+:: Set $HOME to the current dir so msys runs here
+set HOME=%cd%
+
+:: Configure, build, test, and install using `nmake`.
+bash -lc "make"
+if errorlevel 1 exit 1
+bash -lc "make DYNAMIC_ARCH=1 BINARY=$ARCH NO_LAPACK=0 NO_AFFINITY=1 USE_THREAD=1 PREFIX=$LIBRARY_PREFIX install"
```

Comment:
So, I have updated the args here. Enabling `DYNAMIC_ARCH` lets us build for multiple CPUs with different types of available instructions. As a result, we still get AVX instructions when they are available, but if they are not we proceed through a slower route (but don't crash due to missing instructions). I have found the LAPACK part doesn't like to build unless I tell not to not build. I'm not sure if no affinity works well on Windows, but it does on Mac and Linux.

I have enabled threading, but I may disable it on all platforms before this is over. I haven't had crashes from threading using these settings on Mac and Linux (Windows is an open question), but I have seen some performance hits on certain versions. Threading logic in OpenBLAS ends up having to use some heuristics that inevitably breakdown. Personally, I find that doing the threading at a higher level (breaking the problem into pieces and letting the GIL handle this or shipping it off to totally different Python process) can work very nicely in practice and doesn't have nearly as many surprises.